hazelcast/src/test/resources/hazelcast-multicast-plugin.xml

```diff
@@ -32,6 +32,9 @@
             <aws enabled="false"/>
             <discovery-strategies>
                 <discovery-strategy enabled="true" class="com.hazelcast.spi.discovery.multicast.MulticastDiscoveryStrategy">
+                    <properties>
+                        <property name="port">54329</property>
```

Comment:
Hmm, let's say MemberToMemberDiscoveryTest was the test that was interfering. This file is used for both tests. Should we create new file? WDYT?

Reply:
Makes sense 👍